ops/pebble.py

```diff
@@ -381,14 +341,7 @@ def __repr__(self):
 
 
 class ChangeError(Error):
-    """Raised by actions when a change is ready but has an error.
-
-    For example, this happens when you attempt to start an already-started
-    service:
-
-    cannot perform the following tasks:
-    - Start service "test" (service "test" was previously started)
-    """
```

Comment:
Drive by fix: Starting an already-started service now succeeds (start is idempotent). So just remove this comment.